### PR TITLE
load TMPE in asset editor

### PR DESCRIPTION
**Pull request by kianzarrin**
part of #959
I enable TMPE in all game modes. some parts of TMPE (related to non-existing Panels) are disabled if we are not in play mode.

Hint: To make review easier ignore white spaces.
Note: I did not skip the loading of TMPE pathfinder because it's not going to be called anyway.

### CODE REVIEW

File: TLM/TLM/LoadingExtension.cs [hunk 1]
```diff
@@ -169,86 +172,74 @@ public class LoadingExtension : LoadingExtensionBase {
...
-                        return;
-                    }
+            if (BuildConfig.applicationVersion != BuildConfig.VersionToString(
+                    TrafficManagerMod.GAME_VERSION,
+                    false)) {
+                string[] majorVersionElms = BuildConfig.applicationVersion.Split('-');
+                string[] versionElms = majorVersionElms[0].Split('.');
+                uint versionA = Convert.ToUInt32(versionElms[0]);
+                uint versionB = Convert.ToUInt32(versionElms[1]);
+                uint versionC = Convert.ToUInt32(versionElms[2]);
```
Comment (baizsof) [inline TLM/TLM/LoadingExtension.cs:182]: Would kindly advise a better naming conventions such as
(main).(edition).(repair)

or  (Major version).(Minor version).(Revision number) as suggested here

https://softwareengineering.stackexchange.com/questions/3199/what-version-naming-convention-do-you-use/3225#3225

File: TLM/TLM/LoadingExtension.cs [hunk 2]
```diff
@@ -169,86 +172,74 @@ public class LoadingExtension : LoadingExtensionBase {
...
-
-                default: {
-                        Log.Info($"OnLevelLoaded: Unsupported game mode {mode}");
-                        return;
-                    }
+            if (BuildConfig.applicationVersion != BuildConfig.VersionToString(
+                    TrafficManagerMod.GAME_VERSION,
+                    false)) {
+                string[] majorVersionElms = BuildConfig.applicationVersion.Split('-');
+                string[] versionElms = majorVersionElms[0].Split('.');
```
Comment (baizsof) [inline TLM/TLM/LoadingExtension.cs:179]: would kindly raise an improvement idea:
I would use a regex pattern ( for instance: \d+.\d+-\d+ ) to parse the version elements.
In my view, regex parsing could avoid chain calling such as did in 179 line and would also be more readable. 

More about Regex in C#
https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.match?view=netcore-3.1

File: TLM/TLM/LoadingExtension.cs (hunk 1 above)
Comment (kianzarrin) [reply TLM/TLM/LoadingExtension.cs:182]: I didn't change this line. you should ignore white spaces.

File: TLM/TLM/LoadingExtension.cs (hunk 2 above)
Comment (kianzarrin) [reply TLM/TLM/LoadingExtension.cs:179]: again ignore white spaces.

**Pull request merged**